### ISSUE TITLE
New check: JavadocEndCommentDelimiter

### ISSUE THREAD

**Issue by gianmarcoschifone** [checkstyle/checkstyle#20982]
# Description
After https://github.com/checkstyle/checkstyle/issues/19808

The last line of a Javadoc comment contains the end-comment delimiter (`*/`). Unlike the begin-comment delimiter (`/**`), the end-comment delimiter contains only a single asterisk.

Currently there is no check ensuring that the closing delimiter uses exactly one asterisk. As a result, invalid forms such as `**/` may be accepted.

This check should report Javadoc comments whose closing delimiter has more than one asterisk before the slash.

# Config

```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">
        <module name="JavadocEndCommentDelimiter"/>
    </module>
</module>
```

# Example
```java
public class Test {

    /**
     * Valid Javadoc.
     */
    public void valid1() { }

    /** Valid single-line Javadoc. */
    public void valid2() { }

    /**
     * Invalid Javadoc.
     **/ // violation
    public void invalid1() { }

    /** Invalid single-line Javadoc. **/ // violation
    public void invalid2() { }
}
```

# Output
```
Starting audit...
[ERROR] Test.java:/:/: Javadoc closing delimiter should contain exactly one asterisk. [JavadocEndCommentDelimiter]
[ERROR] Test.java:/:/: Javadoc closing delimiter should contain exactly one asterisk. [JavadocEndCommentDelimiter]
Audit done.
Checkstyle ends with 2 errors.
```
